### ISSUE TITLE
Username verification error

### ISSUE THREAD

**Issue by pinkunicorn** [RestyaPlatform/board#2709]
## Expected Behavior
Being able to add an email address to a user that does not have one.

## Current Behavior
Can't save modified user due to username verification issue.

## Steps to Reproduce (for bugs)
1. Open user first_last for edit.
2. Change the email address.
3. Try to save.

## Context
I'm going through my users adding email addresses to all users that have been imported from Trello (due to https://github.com/RestyaPlatform/board/issues/2694) and came upon a user that had a username on the form first_last. When I try to save that user, I get an error saying that the username must be 3-90 characters long and not contain any spaces. The username in question does not violate any of that, but I don't know whether the validation check or the error message is in error.

## Your Environment
* Version used: 0.6.7
* Environment name and version (e.g. OS, PHP, PostgreSQL): CentOS 7, Postgresql 9.6.14

**Comment by sridhar391** [comment]
@pinkunicorn can you please try to reproduce the issue in our demo site with credentials from https://restya.com/board/demo and please share the steps to reproduce the issue in the form of image or video

**Comment by pinkunicorn** [comment]
I created a test user at Trello, "foo_bar", and created a board there with one item. I exported this board to a json file (attached)
[trello.zip](https://github.com/RestyaPlatform/board/files/3371568/trello.zip)

I then logged into the demo restya board as user and imported this json file, which created a board with an item created by "foo_bar". Then I log into the demo restya board as admin and attempt to add an email address for this user, but this is not possible since the validation check on the username field won't let me save this already existing user.

Since the user is valid when importing from Trello and can be used in restyaboard as is (without the email address), it's probably better to adjust the validation check to allow "_" in the username than to disallow Trello import of files containing users that have "_" in their names.
